trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -305,42 +451,147 @@ def VLM(conditions,settings,geometry):
-
-    # Now calculate the coefficients for each wing and in total
+    # Now calculate the coefficients for each wing
     cl_y     = LIFT/CHORD_strip/ES
     cdi_y    = DRAG/CHORD_strip/ES
     CL_wing  = np.add.reduceat(LIFT,span_breaks,axis=1)/SURF
     CDi_wing = np.add.reduceat(DRAG,span_breaks,axis=1)/SURF
-    CL       = np.atleast_2d(np.sum(LIFT,axis=1)/SREF).T
-    CDi      = np.atleast_2d(np.sum(DRAG,axis=1)/SREF).T
-    CM       = np.atleast_2d(np.sum(MOMENT,axis=1)/SREF).T/c_bar
-
     alpha_i = np.hsplit(np.arctan(cdi_y/cl_y),span_breaks[1:])
     
-    return CL, CDi, CM, CL_wing, CDi_wing, cl_y, cdi_y, alpha_i, CP, gamma
\ No newline at end of file
+    # Now calculate total coefficients
+    CL       = np.atleast_2d(np.sum(LIFT,axis=1)/SREF).T          # CLTOT in VORLAX
+    CDi      = np.atleast_2d(np.sum(DRAG,axis=1)/SREF).T          # CDTOT in VORLAX
+    CM       = np.atleast_2d(np.sum(MOMENT,axis=1)/SREF).T/c_bar  # CMTOT in VORLAX
+
+    CYTOT    = np.atleast_2d(np.sum(FY,axis=1)/SREF).T   # total y force coeff
+    CRTOT    = np.atleast_2d(np.sum(RM,axis=1)/SREF).T   # rolling moment coeff (unscaled)
+    CRMTOT   = CRTOT/w_span*(-1)                         # rolling moment coeff
+    CNTOT    = np.atleast_2d(np.sum(YM,axis=1)/SREF).T   # yawing  moment coeff (unscaled)
+    CYMTOT   = CNTOT/w_span*(-1)                         # yawing  moment coeff
+
+    # ---------------------------------------------------------------------------------------
+    # STEP 13: Pack outputs
+    # ------------------ --------------------------------------------------------------------     
+    precision      = settings.floating_point_precision
+    
+    #VORLAX _TOT outputs
+    results = Data()
+    results.CL         =  CL         
+    results.CDi        =  CDi        
+    results.CM         =  CM  
+    results.CYTOT      =  CYTOT
+    results.CRTOT      =  CRTOT
+    results.CRMTOT     =  CRMTOT
+    results.CNTOT      =  CNTOT
+    results.CYMTOT     =  CYMTOT
+    
+    #other SUAVE outputs
+    results.CL_wing    =  CL_wing   
+    results.CDi_wing   =  CDi_wing 
+    results.cl_y       =  cl_y     
+    results.cdi_y      =  cdi_y     
+    results.alpha_i    =  alpha_i  
+    results.CP         =  np.array(CP    , dtype=precision)
+    results.gamma      =  np.array(GAMMA , dtype=precision)
+    
+    return results
+
+# ----------------------------------------------------------------------
+#  CLE rotation effects helper function
+# ----------------------------------------------------------------------
+def compute_rotation_effects(VD, settings, EW_small, GAMMA, len_mach, X, CHORD, XLE, XBAR, 
+                             rhs, COSINP, SINALF, PITCH, ROLL, YAW, STB, RNMAX):
+    """ This computes the effects of the freestream and aircraft rotation rate on 
+    CLE, the induced flow at the leading edge
+    
+    Assumptions:
+    Several of the values needed in this calculation have been computed earlier in
```

Comment:
move some of your statements that are in the comments up, that way they are in our documentation

Reply:
Just address this comment of moving the todo and cleaning up the comments up to this docstring and I'll approve.